kernel/fs_node/src/lib.rs

```diff
@@ -24,6 +24,13 @@ use alloc::sync::{Arc, Weak};
 use memory::MappedPages;
 use io::{ByteReader, ByteWriter, KnownLength};
 
+/// A value to represent the space on the disk (stored in meta-information) for directores
```

Comment:
What is this? I don't understand, a directory doesn't necessarily occupy 4096 bytes. For a directory, you should either calculate the full size of all of its children, or for an easier option, just use something like `--` to show that a size is N/A.

Reply:
Thanks for letting me know (when researching, I found that most linux systems obtains the size of a directory from its metadata, which usually just indicate the directory size as 4096)

However, I will add the -- option for directory size as that might be more clearer and less misleading